### ISSUE TITLE
fxa-mailer repo's config.js for some reason throws a token error during string extraction

### ISSUE THREAD

**Issue by vladikoff** [mozilla/fxa-auth-mailer#114]

**Comment by pdehaan** [comment]
Looks like it really hates that `const` keyword.

### before:
```sh
$ grunt
Running "nunjucks:render" (nunjucks) task
File "templates/new_sync_device.html" created.
File "templates/password_changed.html" created.
File "templates/password_reset.html" created.
File "templates/post_verify.html" created.
File "templates/reset.html" created.
File "templates/unlock.html" created.
File "templates/verification_reminder.html" created.
File "templates/verify.html" created.
File "templates/verify_reminder_first.html" created.
File "templates/verify_reminder_second.html" created.

Running "copy:strings" (copy) task


Running "l10n-extract" task
Fatal error: Unexpected token (5:6)
```

### after:
```sh
$ grunt
Running "nunjucks:render" (nunjucks) task
File "templates/new_sync_device.html" created.
File "templates/password_changed.html" created.
File "templates/password_reset.html" created.
File "templates/post_verify.html" created.
File "templates/reset.html" created.
File "templates/unlock.html" created.
File "templates/verification_reminder.html" created.
File "templates/verify.html" created.
File "templates/verify_reminder_first.html" created.
File "templates/verify_reminder_second.html" created.

Running "copy:strings" (copy) task


Running "l10n-extract" task

Done, without errors.
```

---

### diff:
```sh
$ git diff

diff --git a/config.js b/config.js
index 00e0923..37040cc 100644
--- a/config.js
+++ b/config.js
@@ -2,12 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

-const fs = require('fs')
-const path = require('path')
+var fs = require('fs')
+var path = require('path')

-const convict = require('convict')
+var convict = require('convict')

-const conf = convict({
+var conf = convict({
   env: {
     doc: 'The current node.js environment',
     default: 'prod',
(END)
```

**Comment by pdehaan** [comment]
Not sure if we want to tweak the [.travis.yml](https://github.com/mozilla/fxa-auth-mailer/blob/26ced35d881466fb4b38b093445b18c0c2db9281/.travis.yml#L33) file to run the Grunt `default` task (instead of just `grunt templates`), so it also tests `l10n-extract` on each commit.

**Comment by pdehaan** [comment]
Fixed by #115

**Comment by vladikoff** [comment]
Thanks @pdehaan !!!!!!! :exclamation:
